test/e2e/bundle_e2e_test.go

```diff
@@ -16,6 +20,73 @@ import (
 	"github.com/operator-framework/operator-lifecycle-manager/test/e2e/ctx"
 )
 
+const vpaCRDManifest = `
```

Comment:
can we store this fixture in a `testdata` directory?

Reply:
Ok, that was my original thought but I decided to stick it in the file because I was unsure if the file would be there in the e2e test execution context on prow. Will try that approach.